src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n+1){
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n+2){
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--3 :nth-child(-n+3){
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4){
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--5 :nth-child(-n+5){
+  background-image: url(./images/star-active.svg);
```

Comment:
you are using the same rule for 5 different selectors
I suggest combining these selectors to write the rules once
for example:
```
.selector:nth-child(1),
.selector:nth-child(2) {
  display: block;
}
```